aws/resource_aws_efs_file_system.go

```diff
@@ -233,6 +238,9 @@ func resourceAwsEfsFileSystemRead(d *schema.ResourceData, meta interface{}) erro
 		d.SetId("")
 		return nil
 	}
+    if fs.Endpoint != nil {
```

Comment:
@bflad I added these 3 lines to set the fs.Endpoint. The Travis CI fails on fmt check but I'm not sure why it fails. Could you please advice.